src/engine/storage/HTTPStorageEngine.js

```
@@ -0,0 +1,220 @@
+                throw new ModelNotFoundStorageEngineError(id);
+            }
+            throw error;
+        });
+    }
+
+    /**
+     * Get a model's index data
+     * @param {Model.constructor} constructor
+     * @return Promise<void>
+     */
+    getIndex(constructor) {
```

Biome (1.9.4)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)